docs/framework/network-programming/networkinformation.md

```diff
@@ -20,6 +20,6 @@ The <xref:System.Net.NetworkInformation> namespace enables you to gather informa
 ## See also
 
 - [Network Programming Samples](network-programming-samples.md)
-- [Network Information Technology Sample](https://go.microsoft.com/fwlink/?LinkID=179564)
-- [NetStat Tool Technology Sample](https://go.microsoft.com/fwlink/?LinkID=179562)
-- [Ping Client Technology Sample](https://go.microsoft.com/fwlink/?LinkID=179565)
+- [Network Information Technology Sample](https://archive.msdn.microsoft.com/nclsamples/Wiki/View.aspx?title=Network%20Information)
```

Comment:
I wouldn't change the links if they are going to a broken URL. @davidsh @karelz have these NCL samples been migrated somewhere?